public/sass/elements/_forms.scss

```diff
@@ -3,38 +3,21 @@
 
 // Contents:
 //
-// 1. Helpers
-// 2. Form wrappers
-// 3. Form labels
-// 4. Form hints
-// 5. Form controls
-// 6. Form control widths
+// 1. Form wrappers
+// 2. Form labels
+// 3. Form hints
+// 4. Form controls
+// 5. Form control widths
 
-// 1. Helpers
+// 1. Form wrappers
 // ==========================================================================
 
-// Fieldset is used to group more than one .form-group
-fieldset {
+// .fieldset is used to apply styling to fieldsets and to group more than one .form-group
+.fieldset {
```

Comment:
Absolutely, that's perfectly sensible. We don't want this class applied to anything other than a `<fieldset>`. I'll update it.